src/implementations/vanilla/components/site-header-standardised/site-header-standardised.scss

```diff
@@ -0,0 +1,298 @@
+    padding-bottom: map.get(spacing.$spacing, 'l');
+    padding-top: map.get(spacing.$spacing, 'l');
+  }
+
+  .ecl-site-header-standardised__action {
+    align-items: center;
+  }
+
+  .ecl-site-header-standardised__logo-link {
+    margin-bottom: 0;
+  }
+
+  .ecl-site-header-standardised__logo-image {
+    height: $_logo-height-lg;
+    width: $_logo-width-lg;
+  }
+
+  .ecl-site-header-standardised__login-box {
+    left: auto;
+    margin-top: calc(#{map.get(spacing.$spacing, 'xl')} - 1px);
+    width: auto;
+  }
+
+  .ecl-site-header-standardised__login-toggle,
+  .ecl-site-header-standardised__language-selector {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    font: map.get(typography.$font, 'm');
+    margin: 0;
+    padding: map.get(spacing.$spacing, 's');
+  }
+
+  .ecl-site-header-standardised__login-toggle[aria-expanded='true'] {
+    margin-bottom: -#{map.get(spacing.$spacing, 'xl')};
+    padding-bottom: calc(
+      #{map.get(spacing.$spacing, 's')} + #{map.get(spacing.$spacing, 'xl')}
+    );
+  }
+
+  .ecl-site-header-standardised__login-toggle,
+  .ecl-site-header-standardised__language-icon {
+    .ecl-site-header-standardised__icon {
+      height: map.get(icon.$icon, 'm');
+      margin-right: map.get(spacing.$spacing, 'xs');
+      width: map.get(icon.$icon, 'm');
+    }
+  }
+
+  .ecl-site-header-standardised__login-toggle {
+    .ecl-site-header-standardised__login-arrow {
+      display: block;
+      height: map.get(icon.$icon, 'xs');
+      margin-left: map.get(icon.$icon, 'xs');
+      transform: rotate(0);
+      width: map.get(icon.$icon, 'xs');
+    }
+    &[aria-expanded='false'] {
+      .ecl-site-header-standardised__login-arrow {
+        transform: rotate(180deg);
+      }
+    }
+  }
+
+  .ecl-site-header-standardised__language-code {
+    font: map.get(typography.$font, 'xs');
+    margin-right: map.get(spacing.$spacing, 'xs');
+    transform: translateX(-50%) translateX(-0.2rem) translateY(-50%)
+      translateY(-0.05rem);
+  }
+
+  .ecl-site-header-standardised__search-container {
+    flex-grow: 1;
+    margin-left: map.get(spacing.$spacing, 's');
+  }
+
+  .ecl-site-header-standardised__search-toggle {
+    display: none;
+  }
+
+  .ecl-site-header-standardised__search {
+    background-color: map.get(color.$color, 'white');
+    border-width: 0;
+    box-shadow: none;
+    display: flex;
+    margin: 0;
+    padding: 0;
+    position: relative;
+    top: 0;
+    width: $_search-width-l;
+  }
+
+  .ecl-site-header-standardised__banner-top,
+  .ecl-site-header-standardised__banner {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .ecl-site-header-standardised__banner-top {
+    font: map.get(typography.$font-prolonged, 'm');
+  }
+
+  .ecl-site-header-standardised__banner {
+    font: map.get(typography.$font, '3xl');
+    padding-bottom: map.get(spacing.$spacing, 'l');
+    padding-top: map.get(spacing.$spacing, 'l');
+  }
+}
+
+@include breakpoints.up(xl) {
```

Comment:
Same comment as in the other header component https://github.com/ec-europa/europa-component-library/pull/1772#discussion_r529343715